src/structures/paging/page.rs

```diff
@@ -438,4 +447,23 @@ mod tests {
         }
         assert_eq!(range_inclusive.next(), None);
     }
+
+    #[test]
+    pub fn test_max_page_overflow() {
```

Comment:
Perhaps the test name could mention that it's testing `PageRangeInclusive`.

Reply:
What do you think of `test_page_range_inclusive_overflow`? It's a bit wordy, not sure if that is a concern. But should describe what the test does nicely.

Reply:
Sure, that works.